lib/ocpp/common/websocket/websocket_tls_tpm.cpp

```diff
@@ -717,11 +738,14 @@ void WebsocketTlsTPM::on_writable() {
         return;
     }
 
-    while (false == message_queue.empty()) {
+    while (true) {
```

Comment:
The logic here is not correct. The behavior for libwebsockets differs than other libs. We can't have here a while(true) because that does not mean that the message has been sent to the wire (and we go back to the websocket++ problem)

The correct behavior is:
1. Poll the message
2. If the callback 'on_writable' is called again a next iteration then it means that the message has been 'truly' sent to the wire. In the current case right now it just means that it was polled to the internal buffer of libwebsockets it doesn't mean that it was sent to the wire. The behavior of libwebsocket is that when it's internal buffer is emptied it will by default call 'on_writable' again, and only then we can pop the message from the message queue.